.htaccess

```diff
@@ -53,3 +53,5 @@ Options -Indexes
     php_value session.cookie_lifetime 3600
     php_value session.gc_maxlifetime 3600
 </IfModule>
+
+Header set Access-Control-Allow-Origin "*"
```

Comment:
At some point, we're going to have to think about this one, because it has some implications for security.

Reply:
Yeah this piece is NOT required except to get swagger.io to work with the docker branch.
All the CORS stuff will NOT be included in the actaul pull request.